solr/core/src/test/org/apache/solr/cloud/api/collections/ShardSplitTest.java

```diff
@@ -635,7 +635,7 @@ private void doSplitShardWithRule(SolrIndexSplitter.SplitMethod splitMethod) thr
     log.info("Starting testSplitShardWithRule");
     String collectionName = "shardSplitWithRule_" + splitMethod.toLower();
     CollectionAdminRequest.Create createRequest = CollectionAdminRequest.createCollection(collectionName, "conf1", 1, 2)
-        .setRule("shard:*,replica:<2,node:*");
```

Comment:
Doesn't seem to be an Autoscaling related rule.

Reply:
(whatever that would mean BTW :))

Reply:
Yeah , true . it was the previous rule